### PR TITLE
mpg123: add page

### PR DESCRIPTION
**Pull request by hellojukay**
- [x] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/mpg123.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG Audio Player.
+> More information: <https://www.mpg123.de/>.
+
+- Play mp3 files:
+
+`mpg123 {{path/target_file1.mp3}} {path/target_file2.mp3}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:8]:
```suggestion
`mpg123 {{path/to/file1.mp3}} {{path/to/file2.mp3}} ...`
```

File: pages/linux/mpg123.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG Audio Player.
+> More information: <https://www.mpg123.de/>.
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:4]: Maybe better use manned.org:

```suggestion
> More information: <https://manned.org/mpg123.1>.
```

File: pages/linux/mpg123.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Play mp3 files:
+
+`mpg123 {{path/target_file1.mp3}} {path/target_file2.mp3}}`
+
+- Play a mp3 from stdin:
+
+`mpg123 -`
+
+- Interactive command  to jump forward to the next song:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:14]:
```suggestion
- Interactive command to jump forward to the next song:
```

File: pages/linux/mpg123.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`f`
+
+- Interactive command  to jump back to the beginning for the song:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:18]:
```suggestion
- Interactive command to jump back to the beginning for the song:
```

File: pages/linux/mpg123.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`b`
+
+- Interactive command  to stop or replay:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:22]:
```suggestion
- Interactive command to stop or replay:
```

File: pages/linux/mpg123.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`s`
+
+- Interactive command  to fast forward:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:26]:
```suggestion
- Interactive command to fast forward:
```

File: pages/linux/mpg123.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`.`
+
+- Interactive command  to quit:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:30]:
```suggestion
- Interactive command to quit:
```

File: pages/linux/mpg123.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG Audio Player.
+> More information: <https://www.mpg123.de/>.
+
+- Play mp3 files:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:6]:
```suggestion
- Play the specified mp3 files:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
I think we should mainly focus on a tool CLI instead of it's interactive usage in general case. ;) But I don't want to say that described interactive usage is useless.

File: pages/linux/mpg123.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG Audio Player.
+> More information: <<https://manned.org/mpg123.1>.
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:4]:
```suggestion
> More information: <https://manned.org/mpg123.1>.
```

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/linux/mpg123.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG Audio Player.
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:3]: Maybe this better:

```suggestion
> Console MPEG audio player.
```

File: pages/linux/mpg123.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://manned.org/mpg123.1>.
+
+- Play the specified mp3 files:
+
+`mpg123 {{path/to/file1.mp3 path/to/file2.mp3 ...}}`
+
+- Play a mp3 from stdin:
```
Comment (EmilySeville7cfg) [inline pages/linux/mpg123.md:10]:
```suggestion
- Play the mp3 from stdin:
```

**Review by EmilySeville7cfg — APPROVED** [review]
Everything looks normal. :sunglasses:

File: pages/linux/mpg123.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+> Console MPEG audio player.
+> More information: <https://manned.org/mpg123.1>.
+
+- Play the specified mp3 files:
+
+`mpg123 {{path/to/file1.mp3 path/to/file2.mp3 ...}}`
+
+- Play the mp3 from stdin:
+
+`mpg123 -`
```
Comment (pixelcmtd) [inline pages/linux/mpg123.md:12]: We usually use examples like this (e.g. `tldr llvm-as`):

```suggestion
`cat {{file.mp3}} | mpg123 -`
```

File: pages/linux/mpg123.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
+# mpg123
+
+> Console MPEG audio player.
+> More information: <https://manned.org/mpg123.1>.
```
Comment (blueskyson) [inline pages/linux/mpg123.md:4]:
```suggestion
> More information: <https://manned.org/mpg123>.
```

File: pages/linux/mpg123.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+- Play the specified mp3 files:
+
+`mpg123 {{path/to/file1.mp3 path/to/file2.mp3 ...}}`
+
+- Play the mp3 from stdin:
+
+`cat {{file.mp3}} | mpg123 -`
+
+- Interactive command to jump forward to the next song:
```
Comment (navarroaxel) [inline pages/linux/mpg123.md:14]: We don't mark the hotkeys for terminal apps
```suggestion
- Jump forward to the next song:
```

File: pages/linux/mpg123.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+`f`
+
+- Interactive command to jump back to the beginning for the song:
```
Comment (navarroaxel) [inline pages/linux/mpg123.md:18]:
```suggestion
- Jump back to the beginning for the song:
```

File: pages/linux/mpg123.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+
+- Interactive command to stop or replay:
+
+`s`
+
+- Interactive command to fast forward:
+
+`.`
+
+- Interactive command to quit:
```
Comment (navarroaxel) [inline pages/linux/mpg123.md:30]:
```suggestion
- Quit:
```

File: pages/linux/mpg123.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+
+- Interactive command to jump forward to the next song:
+
+`f`
+
+- Interactive command to jump back to the beginning for the song:
+
+`b`
+
+- Interactive command to stop or replay:
```
Comment (navarroaxel) [inline pages/linux/mpg123.md:22]: Should we specify the current track?
```suggestion
- Stop or replay the current file:
```

**Review by blueskyson — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Thanks! :З

**Pull request merged**